### PR TITLE
Update installation docs to use sudo

### PR DESCRIPTION
**Pull request by stevenGravy**
Sudo (or running as root or a user with privileges) is required for these installation steps.  This makes it consistent with other installation examples in docs.

### CODE REVIEW

**Review by inertial-frame — APPROVED** [review]

**Comment by awly** [comment]
I removed `sudo` from `./install` script calls, when installing from a tarball.
It's basically `curl | sudo bash` with extra steps, which is how codecov got popped.
otherwise LGTM

**Review by awly — APPROVED** [review]

**Comment by awly** [comment]
@klizhentas or @russjones can one of you approve too?

**Comment by stevenGravy** [comment]
> I removed `sudo` from `./install` script calls, when installing from a tarball.
> It's basically `curl | sudo bash` with extra steps, which is how codecov got popped.
> otherwise LGTM

thanks @awly !

File: docs/pages/installation.mdx [hunk 1]
```diff
@@ -15,23 +15,23 @@ up-to-date information.
...
+  ```
   </TabItem>
 
   <TabItem label="Amazon Linux 2/RHEL/Fedora (RPM)">
-    ```bash
-    $ yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-    $ yum install teleport
-    ```
+  ```bash
+  $ sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
```
Comment (klizhentas) [inline docs/pages/installation.mdx:32]: let's remove dollar signs here and below

**Review by klizhentas — CHANGES_REQUESTED** [review]

**Comment by klizhentas** [comment]
@stevenGravy ping, let's remove this PR, and if you would like to push this change, please rebase against master.

**Comment by russjones** [comment]
@stevenGravy If you still want to make these changes, please re-open or create another PR.

**Pull request closed without merging**